src/components/AbstractSelectableComponent.js

```diff
@@ -119,24 +108,20 @@ export default class AbstractSelectableComponent extends React.Component {
     updateViewInfo(this.viewInfo);
   }
 
-  renderLayersMenu() { // eslint-disable-line class-methods-use-this
-    // No-op
-  }
-
   render() {
     const { tool } = this.state;
     const toolProps = {
       setActiveTool: (toolUpdate) => { this.setState({ tool: toolUpdate }); },
       /* eslint-disable react/destructuring-assignment */
       isActiveTool: toolCheck => (toolCheck === this.state.tool),
       /* esline-enable */
```

Comment:
@mccalluc is there a reason for the state destructuring of tool here which is then not used in `isActiveTool` comparison? Sorry, first time looking at this closely.

Reply:
The React docs explain that composition is better than subclassing... but I hadn't read that section when I wrote this, so this abstract superclass has concrete subclasses... Is that what you're asking?

Reply:
Sorry, I think I just got confused with the syntax and eslinting config. The `tool` is destructured in 111 (from `this.state`) and then a comparison is made with `this.state.tool` in  131. I would just change `(toolCheck === this.state.tool)` to `(toolCheck === tool)` and remove the eslint comments unless there is a reason this is explicit. 

If not, we should change `/* esline-enable */` to `/* eslint-enable */` to re-enable linting.

Reply:
Oh, I see what you're asking. It might work, but `isActiveTool` is going to be called at some later point, and we want to be sure that it's referencing the current value in `this`, and not the value when `isActiveTool` was defined.... it might just capture the old value like a closure.

I'd say changing it and seeing what happens might be a learning experience, but it may not work in the end. Thanks for following up.

Reply:
Ok, I see your point. Thank you for the clarification.